.gitignore

```diff
@@ -95,3 +93,11 @@ metastore_db
 # files from R-package source install
 **/config.status
 R-package/src/Makevars
+
+# Visual Studio Code
+/.vscode/
+
+# IntelliJ/CLion
+/.idea/
```

Comment:
I don't use it, but why is an extra `/`?

Reply:
That's what CLion does when you ask it to ignore a directory, it's slightly more precise, a directory at the repository root only. Changed it back in case anyone creates a project in a subdirectory.